package-lock.json

```diff
@@ -1,8 +1,7056 @@
 {
   "name": "capacitor-site",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I would undo these changes. (`git rm package-lock.json`)

It looks like you're using npm 7 and most of the team is still on npm 6.